packages/simcore-sdk/src/simcore_sdk/node_ports_common/filemanager.py

```diff
@@ -225,19 +266,25 @@ async def _upload_file_to_presigned_links(
                 )
             )
         try:
-            results = await logged_gather(*upload_tasks, log=log, max_concurrency=4)
+            results = await logged_gather(
+                *upload_tasks,
+                log=log,
+                # NOTE: when the file object is already created it cannot be duplicated so
+                # no concurrency is allowed in that case

```

Comment:
Does this refer to the opened file from the disk? If that's the case why not have it opened 4 times?

Reply:
if you read it it says: 
```
# NOTE: when the file object is already created it cannot be duplicated so
# no concurrency is allowed in that case
```
When the file object is **already** created